Quadriga/src/main/java/edu/asu/spring/quadriga/aspects/RestAccessAspect.java

```diff
@@ -25,75 +25,75 @@
 @Component
 public class RestAccessAspect {

```

Comment:
please adjust your formatting settings to use the one we use and reformat:
https://diging.atlassian.net/wiki/pages/viewpage.action?pageId=8781898